test/executor.test.js

```diff
@@ -199,10 +218,13 @@ describe('executor', function() {
   it('does not call autoAttach', function() {
     boot.execute(app, dummyInstructions);
 
+    // FIXME(bajtos) "loopback.DataSource" must stay available in loopback
+    var DataSource = loopback.DataSource || loopback.registry.DataSource;
```

Comment:
This is no longer necessary. See https://github.com/strongloop/loopback/commit/3c509199cc9377d9d2c57a036d8442ebaab31257